index.html

```diff
@@ -69,8 +69,11 @@
 <body class="container">
 	<h1 class="mono">sepheo coop</h1>
 	<div id="about">
-		<p>Sepheo is the working title for a project which aims to become a mutual society for self-employed
-			professionals. We aspire to the <a href="https://en.wikipedia.org/wiki/Rochdale_Principles"
+		<p>SEPHEO (Self-Employed People Helping Each Other) Coop is a mutual society for self-employed
+			professionals. We make connections between coops and freelancers who want to work
```

Comment:
> coops

I am wondering if we want to exclude non-coop organisations by phrasing it this way? 

I think it's be nice if we weren't exclusive, and could work with government, charities, or even limited companies. The coop connection could be implicit, or explicit, in that we are a coop and "principle six" applies.

Reply:
Resolved in later commit/comment